docs/influenza-bioinfo-solutions.md

```diff
@@ -0,0 +1,690 @@
+# Influenza Guidance Document
+
+Authors
+=======
+Emily Smith,
+Gültekin Ünal,
+Gloria Kirabo,
+Simeon Hebrew,
+Nishat Tamanna,
+Katherine Figueroa,
+Daniel Akinyemi,
+Gary Van Domselaar
+
+# Introduction
+
+Influenza remains a significant global health concern. Advances in
+bioinformatics have provided unique opportunities for understanding
+and combating this virus through genomic analysis. This document aims
+to guide the public health community through the challenges of
+standardized genomic analyses for Influenza and bring awareness to
+open-source tools that can be used by either beginners or expert
+bioinformaticians. Here, we outline comprehensive approaches
+integrating both theoretical concepts and practical applications.
+
+## Influenza Genomic Structure
+
+The influenza A virus genome is segmented into eight pieces of
+single-stranded negative-sense viral RNAs (vRNAs). Each of these
+segments is characterized by conserved 5′ and 3′ termini,
+segment-specific untranslated regions (UTRs), and typically contains
+one large open reading frame (ORF). These genomic structures are
+crucial for the virus's ability to use host cellular machinery for
+replication and protein synthesis. The viral RNAs are incorporated
+into viral ribonucleoproteins (vRNPs), which consist of multiple
+copies of nucleoprotein (NP) and RNA-dependent RNA polymerase complex (RdRp), composed of PB1, PB2, and PA subunits. This assembly
+facilitates both the transcription of vRNA into viral mRNA, complete
+with a 5′ cap and a 3′ poly(A) tail, and the replication of vRNA
+through a complementary RNA (cRNA) intermediate. M segment of the
+virus coding matrix gene.**  
+**<img src="images/influenza-guidance-fig1.png" style="width:3.95313in;height:3.28467in" />
```

Comment:
This image is not rendering in my preview tab. Needs tags like this:

<p align="center">
  <img src="./images/hiv-bioinfo-solutions-figure-1.svg" class="center">
</p>

Reply:
Mine rendered, but I had to change the link. The path started with media instead of images.

Reply:
Ah, the paths were renamed, but as you said, the images were also not added